src/Microsoft.ML.GenAI.Core/Microsoft.ML.GenAI.Core.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="AutoGen.Core" Version="$(AutoGenVersion)" />
+    <PackageReference Include="Microsoft.Extensions.AI" Version="$(MicrosoftExtensionsAIVersion)" />
```

Comment:
So is the Extensions.AI package going to be a new permanent dependency for this package?

Reply:
Or would it be better to try and make it like an addon package for your gen ai stuff?

Reply:
This should depend on the M.E.AI.Abstractions assembly rather than M.E.AI.